cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -2267,13 +2309,13 @@ instance ToObject PeerSelectionCounters where
             , "activeNonRootPeers" .= numberOfActiveNonRootPeers
             , "activeNonRootPeersDemotions" .= numberOfActiveNonRootPeersDemotions
 
-            , "knownBootstrapPeers" .= numberOfKnownBootstrapPeers
-            , "coldBootstrapPeersPromotions" .= numberOfColdBootstrapPeersPromotions
-            , "establishedBootstrapPeers" .= numberOfEstablishedBootstrapPeers
-            , "warmBootstrapPeersDemotions" .= numberOfWarmBootstrapPeersDemotions
-            , "warmBootstrapPeersPromotions" .= numberOfWarmBootstrapPeersPromotions
-            , "activeBootstrapPeers" .= numberOfActiveBootstrapPeers
-            , "activeBootstrapPeersDemotions" .= numberOfActiveBootstrapPeersDemotions
+            , "knownBootstrapPeers" .= Cardano.viewKnownBootstrapPeers extraCounters
+            , "coldBootstrapPeersPromotions" .= Cardano.viewColdBootstrapPeersPromotions extraCounters
+            , "establishedBootstrapPeers" .= Cardano.viewEstablishedBootstrapPeers extraCounters
+            , "warmBootstrapPeersDemotions" .= Cardano.viewWarmBootstrapPeersDemotions extraCounters
+            , "warmBootstrapPeersPromotions" .= Cardano.viewWarmBootstrapPeersPromotions extraCounters
+            , "activeBootstrapPeers" .= Cardano.viewActiveBootstrapPeers extraCounters
+            , "activeBootstrapPeersDemotions" .= Cardano.viewActiveBootstrapPeersDemotions extraCounters
```

Comment:
Same considerations apply here as for the new tracing system - does that now spam the logs with `(Set peeraddr, Int)` instead of `Int`)?

Reply:
cc. @coot ? :slightly_smiling_face: